pulp_python/app/viewsets.py

```diff
@@ -5,11 +5,11 @@
 import tempfile
 
 from django.db import transaction
-from django_filters.rest_framework import filterset
 from drf_yasg.utils import swagger_auto_schema
 from rest_framework import decorators, status, serializers
 from rest_framework.response import Response
 
+from pulpcore.app.viewsets import BaseFilterSet
```

Comment:
Plugins should never inherit from pulpcore.app, all plugin inheritance should be done through pulpcore.plugin.

I had a look at BaseFilterSet, and I think it provides enough value that we should expose it in pulpcore.plugin.I'll be happy to merge this PR once that is done, and basefilterset is imported from pulpcore.plugin

Reply:
Oh, thank you for your notice. I made the same mistake in these PRs - [pulp_file/pull/106](https://github.com/pulp/pulp_file/pull/106), [pulp_ansible/pull/44](https://github.com/pulp/pulp_ansible/pull/44). However they've been already merged with the master. So I opened a new PR for pulpcore where ```BaseFilterSet``` is exposed - [pulp/pull/3576](https://github.com/pulp/pulp/pull/3576). For this PR I added new commit and for ```pulp_file``` and ```pulp_ansible``` plugins I opened new PRs - [pulp_file/pull/107](https://github.com/pulp/pulp_file/pull/107),  [pulp_ansible/pull/49](https://github.com/pulp/pulp_ansible/pull/49). I'm sorry for the complications.